Codecademy/gamut
# fix: update Java, JS, and Python icons to latest streamline versions

### Overview



- update Java, JS, and Python icons



Noticed on the new skills dashboard that the old Java, JS, and Python icons are skinnier than the skill icons that I added in #2884. Updating them to the the newest streamline versions in this PR. 

![Screenshot 2024-06-10 at 12 17 59 PM](https://github.com/Codecademy/gamut/assets/65558802/95baf433-739a-4e59-87b2-271179002287)

### PR Checklist


#### Testing Instructions


For some reason, there was no preview generated for this PR, so I guess you have to test locally.
Once you have storybook running locally, checkout the Skills section in the Regular icons page and ensure that you see the updated icons!

Here's what they look like for me!

<img width="1256" alt="Screenshot 2024-06-12 at 9 13 28 AM" src="https://github.com/Codecademy/gamut/assets/65558802/d10c1d52-0b5d-48bb-b863-13b6f442122e">
